lib/Backend/BackendUtils.cpp

```diff
@@ -123,6 +131,109 @@ bool isInput(const Placeholder *PH) {
   return false;
 }
 
+bool isOutput(const Placeholder *PH, const IRFunction &F) {
```

Comment:
Please add a Doxygen comment.

Reply:
> Please add a Doxygen comment.

I moved them to the `BackendUtils.h` . Should I add another one in the cpp file?

Reply:
Oh, I missed it. No, it is fine then.